int32 arm_position
---
bool success